src/main/java/core/basesyntax/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Draw {

```

Comment:
Interface name was not fixed:
![image](https://user-images.githubusercontent.com/117547096/205390705-93d206b5-ea46-4e23-a7e9-086e25225f87.png)